maven-release-manager/src/main/java/org/apache/maven/shared/release/exec/ForkedMavenExecutor.java

```diff
@@ -106,6 +106,10 @@ public void executeGoals(
 
             cl.addEnvironment("MAVEN_TERMINATE_CMD", "on");
 
+            if (releaseEnvironment.getJavaHome() != null) {
+                cl.addEnvironment("JAVA_HOME", releaseEnvironment.getJavaHome().getAbsolutePath());
```

Comment:
I'd expect that always an absolute file is provided, no?

Reply:
Well, in theory the source is decoupled from the release enviroment, so the value could be `null`.

Reply:
Since Maven home does the same to absolute path, let's keep it consistent.